config/dev.exs

```diff
@@ -32,6 +32,10 @@ config :logger, :console, format: "[$level] $message\n"
 # and calculating stacktraces is usually expensive.
 config :phoenix, :stacktrace_depth, 20
 
+# Configure CalendarService
+config :changelog, Changelog.CalendarService,
+  adapter: Changelog.CalendarService
```

Comment:
Yeah, I totally agree with you. Also, another thing that add more confusion is the location of the different implementation of the services (i.e. GoogleCalendarService in Services and CalendarService in data/episode). That was one the thing that I really don't like. So thank you for the feedback.

I would like to end up by providing three explicit different implementation for the services and put all together in the same location:

- `services/google_calendar_service.ex` used for the production
- `services/dummy_calendar_service.ex` used for the development
- `services/mock_calendar_service.ex` used for the test

I think in this case it will be more clear that we may have different implementation of the same service! What do you think?